sdk/openai/azure-ai-openai/src/samples/java/com/azure/ai/openai/ReadmeSamples.java

```diff
@@ -56,6 +57,22 @@ public void createAsyncClientKeyCredential() {
         // END: readme-sample-createAsyncClientKeyCredential
     }
 
+    public void createNonAzureSyncClientWithApiKey() {
+        // BEGIN: readme-sample-createNonAzureOpenAISyncClientApiKey
+        OpenAIClient client = new OpenAIClientBuilder()
+            .credential(new NonAzureOpenAIKeyCredential("{key}"))
+            .buildClient();
+        // END: readme-sample-createNonAzureOpenAISyncClientApiKey
+    }
+
+    public void createNonAzureAsyncClientWithApiKey() {
+        // BEGIN: readme-sample-createNonAzureOpenAIAsyncClientApiKey
+        OpenAIAsyncClient client = new OpenAIClientBuilder()
+            .credential(new NonAzureOpenAIKeyCredential("{key}"))
```

Comment:
nitpick: I would replace `{key}` with `{openai-secret-key}` just for the sake of further clarity. I believe that's how those values are named in the user portal of OpenAI.